lightning/src/ln/peer_handler.rs

```diff
@@ -138,6 +138,14 @@ impl Peer {
 			InitSyncTracker::NodesSyncing(_) => true,
 		}
 	}
+
+	fn should_forward_node(&self, node_id: PublicKey) -> bool {
+		match self.sync_status {
+			InitSyncTracker::NoSyncRequested => true,
+			InitSyncTracker::ChannelsSyncing(_) => false,
+			InitSyncTracker::NodesSyncing(pk) => pk < node_id,
```

Comment:
my lacking Rust knowledge probably, but why is the operator `<`?

Reply:
The idea of this code is that we track where we are in the graph-forwarding process when a peer requests a full dump from us. That way, any time our send buffer to them is empty, we can get the next message to send from the Router and push it onto the sending stack. Here we're checking if we've already sent some part of the graph as we should only send *updates* for that part of the graph after we've sent the original send.

Reply:
Interesting. Also interesting to use pubkey ordering for that. It should probably be in a comment somewhere. Or it already is and I haven't seen it.

Reply:
Good point, its also assume that user-implemented bits do that, so added a commit to describe the requirement.